src/context/walletContext/walletContext.tsx

```diff
@@ -1,16 +1,14 @@
 'use client'
```

Comment:
thought: we expect this context to be available everywhere right? I find the organization of src > context > walletContext a bit confusing. Especially when we have components > Global too. It's  a bit hard to navigate so many folders, I think we might want to dedicate some time to restructuring this

Reply:
thought 2: I think it was nice to have a high level overview comment explaining the goal of this file / WalletProvider. Sad to see it gone

Reply:
Hmmmm I think the distintion between components and context is useful (even if in the end they are all components). I know that a context will have a state available everywhere and components will be reusable UI pieces. (I do agree that we have many folders and sometimes is hard to navigate, if you ask me to write a path to a component I am working on I wont be able to do it)